libindy/src/commands/anoncreds/tails.rs

```diff
@@ -77,10 +77,10 @@ pub fn store_tails_from_generator(service: Rc<BlobStorageService>,
                                   rtg: &mut RevocationTailsGenerator) -> Result<(String, String), CommonError> {
     debug!("store_tails_from_generator >>> writer_handle: {:?}",writer_handle);
 
-    let blob_handle = service.create_blob(writer_handle)?;
-
     let mut hasher = sha2::Sha256::default();
 
+    let blob_handle = service.create_blob(writer_handle, hasher)?;
```

Comment:
In my understanding at this moment `Copy` will occurs for `hasher`. As result code in current PR is equivalent to original.

Reply:
it turns out that the hasher in tails.rs is superfluous and just a waste of time. 
New PR to follow soon